### PR TITLE
Tabular data example

### PR DESCRIPTION
**Pull request by pjforde1978**
Just to be clear, this is almost entirely based on @andrewmcodes' foundation.

I have added filtering and pagination. Tailwind styles had to be converted to Bootstrap 4.

I also am including a lot of proposed cosmetic TLC for the layout.

This PR contains a migration and seed data which needs to be generated.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -19,6 +19,8 @@ gem "sidekiq", "~> 6.0"
 gem "stimulus_reflex", "~> 2.1"
 gem "valid_email", "~> 0.1"
 gem "webpacker", "~> 4.0"
+gem "kaminari", "~> 1.1.1"
```
Comment (hopsoft) [inline Gemfile]: I propose that we switch to the pagy gem as it's more performant. https://github.com/ddnexus/pagy

Comment (pjforde1978) [reply Gemfile]: First I’ve ever heard of it!

**Comment by pjforde1978** [comment]
Let me see what I can do.

**Comment by pjforde1978** [comment]
Also, I’m going to move the db/seeds.rb into a rake task.

**Comment by andrewmcodes** [comment]
You’re amazing @pjforde1978

**Pull request merged**